packages/foreman/nodejs-datatables.net-bs/nodejs-datatables.net-bs.spec

```diff
@@ -4,18 +4,16 @@
 Name: nodejs-%{npm_name}
 Version: 1.10.16
 Release: 1%{?dist}
-Summary: DataTables for jQuery with styling for Bootstrap 3
+Summary: DataTables for jQuery with styling for [Bootstrap 3](http://getbootstrap.com)
```

Comment:
The old one was better since RPMs don't implement markdown. npm2rpm should probably strip it. Summary extraction from description isn't that great yet.